webpack/scenes/ContentViews/Table/ContentViewsTable.js

```diff
@@ -144,24 +177,19 @@ const ContentViewTable = ({ response, status, error }) => {
       <React.Fragment>
         <CopyContentViewModal cvId={actionableCvId} cvName={actionableCvName} show={copy} setIsOpen={setCopy} aria-label="copy_content_view_modal" />
       </React.Fragment>
+      {isPublishModalOpen &&
```

Comment:
Using conditional rendering for modals prevents the modal opening / closing animations provided by Patternfly. Instead you should use the `isOpen` prop of the Patternfly `Modal` component.

Reply:
I tried this and this causes all sorts of issues which the tests also don't like..The component actually renders but remains closed, makes all the dispatch calls and everything. Will have to rely on conditional rendering to avoid that.